cluster/gce/config-default.sh

```diff
@@ -245,8 +245,17 @@ if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
 fi
 
 # Admission Controllers to invoke prior to persisting objects in cluster
-# If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota
+ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority
+
+# Optional: use a default, fairly open PodSecurityPolicy
```

Comment:
what is the intent behind enabling this with an open policy? GCE will have an enormously difficult time tightening it if it does it this way.

Reply:
This comment was out of date.

Reply:
This comment was out of date.